celesta-sql/src/main/java/ru/curs/celesta/score/Grain.java

```diff
@@ -22,12 +21,15 @@
 
 import ru.curs.celesta.CelestaException;
 import ru.curs.celesta.DBType;
+import ru.curs.celesta.score.io.Resource;
 
 /**
  * Grain.
  */
 public final class Grain extends NamedElement {
 
+    private static final Pattern NATIVE_SQL = Pattern.compile("--\\{\\{(.*)--}}", Pattern.DOTALL);
```

Comment:
This is suboptimal regexp for native sql as "greedy dot" can be catastrophically slow (see [this talk](https://www.youtube.com/watch?v=pLKrKFy1t1Y) when you have time).

To write the optimal regexp for native sql was a non-trivial task, the right solution (in JavaCC syntax) is here: https://github.com/CourseOrchestra/celesta/blob/413485c2517f1d9357307f5b8ab1f7aa868babaa/celesta-sql/src/main/javacc/CelestaParser.jj#L142

(and this is [how I derived it](https://twitter.com/inponomarev/status/956663963814645761), if you're curious).

The only aim of this regexp is to strip `--{{` `--}}` from the code. Maybe `startswith`, `endswith` and `substring` will do better.

However, this is not your code, you just  moved it and maybe this whole comment is not relevent to your PR :-)